packages/tools/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Cornerstone3D Tools",
   "main": "dist/umd/index.js",
   "types": "dist/esm/index.d.ts",
-  "module": "dist/esm/index.js",
+  "module": "src/index.ts",
```

Comment:
this should not be src, there should not be any src in the package json for publish

Reply:
It works fine that way as a minimal change, now that the tsconfig is published, AND the source is published - otherwise, I don't see how to debug in both linked and unlinked versions - if you have another suggestion to enable that, please let me know.